fdb-record-layer-spatial/src/test/java/com/apple/foundationdb/record/spatial/geophile/GeophileQueryTest.java

```diff
@@ -114,31 +114,27 @@ protected void loadCities(RecordMetaDataHook hook, int minPopulation) throws Exc
                 String line;
                 int count = 0;
                 while ((line = reader.readLine()) != null) {
-                    try {
```

Comment:
I messed something up, and it was swallowing the exception here, and the tests would just fail because there were no cities loaded. Now it will fail in setup if it can't load the cities.